README.md

```diff
@@ -17,10 +18,22 @@ To start the tests, depending of your OS run:
 - Windows
     1. Install the [AltTesterDesktop](https://alttester.com/app/uploads/AltTester/desktop/AltTesterDesktopPackageWindows__v2.0.1.zip), then open it. 
     2. Create a folder `TrashCatWindows` under `App`. The app is provided at https://alttester.com/app/uploads/AltTester/TrashCat/TrashCatStandAlone2_0_1.zip and needs to be included unzipped under the App/TrashCatWindows/ folder.
+    3. run `./start_tests_Windows.sh` in your terminal
 
-`./start_tests_Windows.sh`
 This script will:
 
 - start the app on your device
 - run the tests
 - stop the app after the test are done
+
+## Allure
+### Run the tests:
+The same setup as in case of windows, but in the thirst step, use the following command  `start_tests_Windows_with_Allure.sh` instead of `./start_tests_Windows.sh`
```

Comment:
```suggestion
The same setup as in the case of Windows, but in the third step, use the following command  `start_tests_Windows_with_Allure.sh` instead of `./start_tests_Windows.sh`
```